src/lib.rs

```diff
@@ -485,47 +484,33 @@ impl Default for PostReaction {
     }
 }
 
-/// Poll data raw storage, each bit represent an item selected.
-// pub struct PollData<ThreadId> {
-//     pub raw: u128,
-//     pub id: ThreadId,
-// }
-
-// /// Some auxiliary method for PollData
-// impl<ThreadId> PollData<ThreadId> {
-//     pub poll_existed(&self, thread: Thread) -> bool {
-//         self.id == thread.id && thread.poll.is_some()
-//     }
-//     pub poll_data_is_valid(&self, poll: Poll<Timestamp, PollItemLength>) -> bool {
-
-//     }
-// }
-
-// pub type PollData = u128;
-// pub type PollItemLength = u8;
-
-/// Represents a poll, each item's description not included.
-// #[cfg_attr(feature = "std", derive(Serialize, Deserialize, Debug))]
-// #[derive(Encode, Decode)]
-// pub struct Poll<T: Trait> {
-//     /// description text for poll
-//     poll_description: Vec<u8>,
+/// Represents all poll alternatives and vote count for each one
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize, Debug))]
+#[derive(Encode, Decode, Clone, PartialEq, Eq)]
+pub struct PollAlternative {
+    /// Alternative description
+    alternative_text: Vec<u8>,
 
-//     /// timestamp of poll start
-//     start_time: Timestamp<T>,
+    /// Vote count for the alternative
+    poll_count: u32,
```

Comment:
Can you rename to be more clear, like `vote_count`.

Reply:
accepted.